aggregator/handler/forward.go

```diff
@@ -53,6 +56,34 @@ type tryConnectFn func(addr string) (*net.TCPConn, error)
 type dialWithTimeoutFn func(protocol string, addr string, timeout time.Duration) (net.Conn, error)
 type writeToConnFn func(conn *net.TCPConn, data []byte) (int, error)
 
+type forwardHandlerMetrics struct {
+	connectTimeouts  tally.Counter
+	connectSuccesses tally.Counter
+	enqueueErrors    tally.Counter
+	enqueueSuccesses tally.Counter
+	currentDropped   tally.Counter
+	oldestdDropped   tally.Counter
+	writeSuccesses   tally.Counter
+	writeErrors      tally.Counter
+	openConnections  tally.Gauge
+	queueSize        tally.Gauge
+}
+
+func newForwardHandlerMetrics(scope tally.Scope) forwardHandlerMetrics {
+	return forwardHandlerMetrics{
+		connectTimeouts:  scope.Counter("connect-timeouts"),
+		connectSuccesses: scope.Counter("connect-successes"),
+		enqueueErrors:    scope.Counter("enqueue-errors"),
+		enqueueSuccesses: scope.Counter("enqueue-successes"),
+		currentDropped:   scope.Counter("current-dropped"),
```

Comment:
Actually it's not the latest, rather the current buffer is the oldest, i think i'll stick to this.